src/components/structures/ScrollPanel.js

```diff
@@ -51,7 +52,15 @@ module.exports = React.createClass({
 
         /* onFillRequest(backwards): a callback which is called on scroll when
          * the user nears the start (backwards = true) or end (backwards =
-         * false) of the list
+         * false) of the list.
+         *
+         * This should return true if the pagination was successful, or false if
+         * there is no more data in this directon (at this time) - which will
```

Comment:
These aren't actually opposites. I could viably return true (pagination successful) and have no more data in this direction (reached the start of the room). Can you please clarify which you mean? I think you mean "return true if you have more data in this direction, else false."